format/FlightSql.proto

```diff
@@ -867,15 +867,15 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
-enum DATE_SUBCODE {
-  UNKNOWN_SUBCODE = 0;
+enum DateSubcode {
+  SQL_DATETIME_SUBCODE_UNKNOWN = 0;
   SQL_CODE_DATE = 1;
   SQL_CODE_TIME = 2;
   SQL_CODE_TIMESTAMP = 3;
 }
```

Comment:
- Should the enum be named SqlDatetimeSubcode?
- Should the other enum variants be renamed SQL_DATETIME_SUBCODE_...?

Reply:
Changed the enum and and I prefixed the values

Reply:
Changed the enum  and I prefixed the values with SQL_DATETIME_SUBCODE_